src/lua/msgpack.c

```diff
@@ -72,6 +74,8 @@ struct luamp_object {
 	const char *data;
 	/** Pointer to the end of msgpack data. */
 	const char *data_end;
+	/** Reference to decoded MsgPack data. */
```

Comment:
Nit: From the comment it's unclear what kind of reference it is, what it equals to initially, and when it's set. Please clarify these points.